internal/templates/common/yamls/deployment.yaml

```diff
@@ -48,6 +48,10 @@ spec:
           {{- if $process.containerPorts }}
           ports:
 {{ $process.containerPorts | toYaml | indent 10 }}
+          {{- end }}
+          {{- if $process.extra.volumes }}
+          volumes:
+{{ $process.extra.volumes | toYaml | indent 12 }}
```

Comment:
according to documentation https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec, we should have `volumes`  under `spec` section:

```yaml
spec:
  containers:
  - name: web
    volumeMounts:
      - name: name-1
        mountPath:
  volumes:
  - name: test-volume
    awsElasticBlockStore:
      fsType: ext4
      volumeID: volume-id

```

Reply:
Moved volumes to the proper location.